litert/vendors/intel_openvino/BUILD

```diff
@@ -16,3 +16,7 @@
 package(
     default_visibility = ["//litert:__subpackages__"],
 )
+
+exports_files([
```

Comment:
Minor: Please consider providing this via a common `cc_library`, e.g. similar to this https://github.com/google-ai-edge/LiteRT/blob/e833d73e671173e016314de2180555a62c8b4734/litert/vendors/qualcomm/BUILD#L30